messaging/WFLY-11293_add_attribute_to_indicate_backup_is_synchronized.adoc

```diff
@@ -0,0 +1,57 @@
+= Add attribute to model which will indicate that backup is synchronized with live server
+:author:            Emmanuel Hugonnet
+:email:             ehugonne@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:          messaging,jms,ha,cluster
+:idseparator:       -
+
+== Overview
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/WFLY-11293[WFLY-11293]
+
+=== Related Issues
+
```

Comment:
Please add https://issues.jboss.org/browse/EAP7-884 here.

Reply:
Hum those are community docs. i don't think those EAP7 issues should be in there

Reply:
We always mentioned related EAP7 issues here. For example:
https://github.com/wildfly/wildfly-proposals/blob/master/messaging/WFLY-10326_messaging_global_resource_usage.adoc
https://github.com/wildfly/wildfly-proposals/blob/master/messaging/WFLY-11829_artemis_clusters_with_http_load_balancer.adoc
We always mentioned related EAP7 issues here.